### PR TITLE
update cli actions & fix batchbuilder

### PR DESCRIPTION
**Pull request by invocamanman**

### CODE REVIEW

File: rollup-cli/src/actions/offchain/send.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ async function send(UrlOperator, idTo, amount, walletJson, password, tokenId, us
                 };
                 walletRollup.signRollupTx(transaction); // sign included in transaction
                 const parsetransaction = stringifyBigInts(transaction);// convert bigint to Strings
```
Comment (krlosMata) [inline rollup-cli/src/actions/offchain/send.js]: We should use lowerCamelCase naming convention:
`parsetransaction` to `parseTransaction`

File: rollup-cli/src/actions/onchain/depositOnTop.js [hunk 2]
```diff
@@ -28,8 +28,8 @@ async function depositOnTop(urlNode, addressSC, balance, tokenId, walletJson, pa
 
     try {
         return new Promise(((resolve, reject) => {
-            axios.get(`${UrlOperator}/offchain/info/${walletBaby.publicKey[0].toString()}/${walletBaby.publicKey[1].toString()}`)
-                .then(async (response) => {
+            apiOperator.getInfoByAxAy(walletBaby.publicKey[0].toString(), walletBaby.publicKey[1].toString())
+                .then(async (response) => { // actualizar post,
```
Comment (krlosMata) [inline rollup-cli/src/actions/onchain/depositOnTop.js]: delete comments :smile:

File: rollup-cli/tools/helpers/sendBot.js [hunk 3]
```diff
@@ -90,7 +90,7 @@ async function send(UrlOperator, idTo, amount, walletJson, password, tokenId, us
                 if (codeWrongTransaction === 11) { // -sign don't matchthe Tx
                     transaction.toIdx += 1;
                 }
-                axios.post(`${UrlOperator}/offchain/send`, parsetransaction)
+                apiOperator.sendOffChainTx(parsetransaction)
```
Comment (krlosMata) [inline rollup-cli/tools/helpers/sendBot.js]: We should use lowerCamelCase naming convention:
`parsetransaction` to `parseTransaction`

Comment (invocamanman) [reply rollup-cli/tools/helpers/sendBot.js]: thanks ^^

**Pull request merged**